From: Sandro Tosi <morph@debian.org>
Date: Sun, 10 Jan 2016 22:16:03 +0000
Subject: Do not fail if we cannot import mingw32

This is required for the Debian version helper, which reads core.setup_common and
executes it, without using all the Python import machinery. This way we will
skip the requirement to have the whole numpy module working while we only care
(at this point) for the API/ABI versions.

Forwarded: not-needed
---
 numpy/core/setup_common.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/numpy/core/setup_common.py b/numpy/core/setup_common.py
index b5bc0de..29f7b05 100644
--- a/numpy/core/setup_common.py
+++ b/numpy/core/setup_common.py
@@ -4,7 +4,10 @@ import pathlib
 import sys
 import textwrap
 
-from numpy.distutils.misc_util import mingw32
+try:
+    from numpy.distutils.misc_util import mingw32
+except:
+    mingw32 = lambda *args, **kwargs: None
 
 
 #-------------------
